sdk/src/azure/platform/az_win32.c

```diff
@@ -23,3 +23,72 @@ AZ_NODISCARD az_result az_platform_sleep_msec(int32_t milliseconds)
   Sleep(milliseconds);
   return AZ_OK;
 }
+
+AZ_NODISCARD az_result az_platform_get_random(int32_t* out_random)
+{
+  _az_PRECONDITION_NOT_NULL(out_random);
+  *out_random = 0;
+  return AZ_ERROR_DEPENDENCY_NOT_PROVIDED;
+}
+
+AZ_NODISCARD az_result az_platform_timer_create(
+    az_platform_timer* timer,
+    az_platform_timer_callback callback,
+    void* sdk_data)
+{
+  (void)timer;
+  (void)callback;
+  (void)sdk_data;
+
+  return AZ_ERROR_DEPENDENCY_NOT_PROVIDED;
+}
+
+void az_platform_critical_error()
+{
+  while (1)
+  {
+  }
+}
+
+AZ_NODISCARD az_result az_platform_timer_start(az_platform_timer* timer, int32_t milliseconds)
+{
+  (void)timer;
+  (void)milliseconds;
+
+  return AZ_ERROR_DEPENDENCY_NOT_PROVIDED;
```

Comment:
Just checking if it's intentional to not have these implemented at this time.

Reply:
It is, we are currently focusing on the Linux posix scenario, but will get back to win32 later.